src/wasm/asm-wasm-builder.cc

```diff
@@ -551,37 +752,47 @@ class AsmWasmBuilderImpl : public AstVisitor {
 #undef SIGNED_BINOP
 #undef NON_SIGNED_BINOP
 
-  int TypeIndexOf(Expression* left, Expression* right, bool ignore_sign) {
-    int left_index = TypeIndexOf(left, true);
-    int right_index = TypeIndexOf(right, true);
-    if (left_index < 0)
+  enum TypeIndex {
+    INT_32 = 0,
```

Comment:
Sorry can you name this as per the style guide:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names